pino.js

```diff
@@ -35,6 +36,10 @@ var defaultOptions = {
   enabled: true,
   messageKey: 'msg'
 }
+var defaultBaseLog = {
```

Comment:
at this point the term "baseLog" has changed conceptually, for historical reasons and to save confusion we should name it differently, even internally 

can we please call this `defaultBase`